gradle-sls-packaging/src/main/java/com/palantir/gradle/dist/service/gc/GcProfile.java

```diff
@@ -102,21 +103,42 @@ public final void newRatio(int newerRatio) {
     // Match the MaxGCPauseMillis case
     @SuppressWarnings("AbbreviationAsWordInName")
     class Hybrid implements GcProfile {
+        // Applies to jdks older than 21:
         // We use 500ms by default, up from the JDK default value of 200ms. Using G1, eden space is dynamically
         // chosen based on the amount of memory which can be collected within the pause time target.
         // Higher pause target values allow for more eden space, resulting in more stable old generation
         // in high-garbage or low-gc-thread scenarios. In the happy case, increasing the pause target increases
         // both throughput and latency. In degenerate cases, a low target can cause the garbage collector to
         // thrash and reduce throughput while increasing latency.
```

Comment:
This comment is slightly misleading as its not the "default" and it might help to also throw in some information about why this is different surrounding the 21 boundary given that we were using `G1GC` before and after the upgrade.

Reply:
Good call, I've replaced it with something that provides better context.